Dockerfile.hinfo

```diff
@@ -1,9 +1,14 @@
 ARG OBS_LSST_VERSION=w_2024_21
 FROM lsstsqre/centos:7-stack-lsst_distrib-${OBS_LSST_VERSION}
 USER lsst
+
 RUN source loadLSST.bash && mamba install aiokafka httpx
-RUN source loadLSST.bash && pip install kafkit
-COPY python/lsst/consdb/hinfo.py python/lsst/consdb/utils.py ./hinfo/
+RUN source loadLSST.bash && pip install kafkit aiokafka httpx
```

Comment:
I tried to do this using only mamba, but it wouldn't cooperate. I think it's because we have to rely on newer versions of aiokafka that have not made their way to conda. My understanding is generally you can get away with mixing pip and conda as long as you never use conda again once you've used pip.